src/lesson4/Calculator.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * @author ann
+ * @data 25.09.22
+ */
+package lesson4;
+
+import java.util.Scanner;
+
+public class Calculator {
+    static boolean askExit = true;
+    static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        double firstNumber = getNumber();
+        double secondNumber = getNumber();
+        String operator = getOperator();
+        calculate(firstNumber, secondNumber, operator);
+        do {
+            askExit = askExit();
+        }while(askExit);
```

Comment:
you can call askExit() method inside while(askExit()) and in this case, you don't need var askExit. naming form method and var is bad